dawn/renderer/utils/sagas.js

```diff
@@ -248,12 +253,13 @@ function* ansibleGamepads() {
     // navigator.getGamepads always returns a reference to the same object. This
     // confuses redux, so we use assignIn to clone to a new object each time.
     const newGamepads = _.assignIn({}, navigator.getGamepads());
-    if (_needToUpdate(newGamepads)) {
+    if (_needToUpdate(newGamepads) || Date.now() - timestamp > 1000) {
```

Comment:
I hope that this one-second throttling does not affect Dawn-runtime communication initiated due to a change in the robot state.